bin/run.js

```diff
@@ -1,10 +1,25 @@
 #!/usr/bin/env node
 
-var call = require('../lib/index').call;
-require("babel/register");
+var config = require('pkg-config')('runjs');
+var call   = require('../lib/index').call;
+
+if (config && config['require-hook']) {
+  require(config['require-hook']);
+}
+else {
+
+    try {
+        require('babel-register');
+    } catch() {
+        try {
+            require('babel/register');
+        } catch() {}
```

Comment:
If we plan to support es6 in the runfile by default I would remove the last try/catch or throw a custom error in the last catch